ADR/0022-secret-mgmt-for-user-workloads.md

```diff
@@ -386,9 +386,9 @@ status:
     type: DataObtained
 ```
 ### Internal RemoteSecrets
-The RemoteSecret Custom Resource (CR) is a crucial implementation detail of "Secret Management For User Workloads". It is also utilized in other features within RHTAP, such as "pull" secrets, among others.
-To differentiate RemoteSecrets that are provided by end-users from RemoteSecrets that are generated by RHTAP's components, a label `appstudio.redhat.com/internal: true` is introduced. The presence of this label signifies that the RemoteSecret is intended for internal usage, and any unauthorized changes to such secrets may result in unpredictable outcomes.
-This label helps to identify secrets that are meant for internal purposes, preventing accidental or unauthorized modifications that could disrupt the functioning of RHTAP's components.
+The RemoteSecret Custom Resource (CR) is a crucial implementation detail of "Secret Management For User Workloads". It is also utilized in other features within Konflux, such as "pull" secrets, among others.
```

Comment:
Isn't it already deprecated and deleted?

Reply:
This is just updating the text for the old ADR. We should have a new ADR recording that RemoteSecret is no longer used.